symphony/lib/boot/func.utilities.php

```diff
@@ -8,32 +8,32 @@
  * Redirects the browser to a specified location. Safer than using a
  * direct header() call
  *
- *  @param string $url
+ *  @param string $location
+ *  @param string $status (optional)
```

Comment:
This may be problematic as the input is not sanitised or checked to be a valid redirect HTTP code. I like the dumbness and simplicity of the redirect function so it's probably ok to say as is. Just something to highlight.